Source/Renderer/ComputeEngine.js

```diff
@@ -0,0 +1,194 @@
+/*global define*/
+define([
+        '../Core/BoundingRectangle',
+        '../Core/Color',
+        '../Core/ComponentDatatype',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/Geometry',
+        '../Core/GeometryAttribute',
+        '../Core/PrimitiveType',
+        '../Shaders/ViewportQuadVS',
+        './BufferUsage',
+        './ClearCommand',
+        './DrawCommand',
+        './Framebuffer',
+        './RenderState',
+        './ShaderProgram',
+        './VertexArray'
+    ], function(
+        BoundingRectangle,
+        Color,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        DeveloperError,
+        Geometry,
+        GeometryAttribute,
+        PrimitiveType,
+        ViewportQuadVS,
+        BufferUsage,
+        ClearCommand,
+        DrawCommand,
+        Framebuffer,
+        RenderState,
+        ShaderProgram,
+        VertexArray) {
+    "use strict";
+
+    /**
+     * @private
+     */
+    var ComputeEngine = function(context) {
+        this._context = context;
+        this._viewportQuadVertexArray = undefined;
+    };
+
+    function createViewportQuadVertexArray(computeEngine) {
```

Comment:
Perhaps rename to `getViewportQuadVertexArray` since it doesn't create it on every call.